src/test_driver/openiotsdk/unit-tests/test_components.txt

```diff
@@ -13,6 +13,7 @@ PlatformTests
 RawTransportTests
 RetransmitTests
 TestShell
+TracingTests
```

Comment:
we did not have these in nl to start with, I think we can skip them here as well for now.

Reply:
Removed.